opentutor_classifier/opentutor_classifier/__init__.py

```diff
@@ -264,6 +264,7 @@ def to_dict(self) -> dict:
 @dataclass
 class TrainingConfig:
     shared_root: str = "shared"
+    property = {"TRAIN_QUALITY": 1}
```

Comment:
let's call this `properties` and make `TRAIN_QUALITY` a constant so it can be imported elsewhere (compiler checking)

```python
from typing import Final

PROP_TRAIN_QUALITY: Final[str] = "TRAIN_QUALITY"

def get_train_quality_default() -> int:
  return os.environ.get("TRAIN_QUALITY_DEFAULT", 1)
```